changelog

```diff
@@ -1,3 +1,395 @@
+          * +40 cargo space (90 -> 130)
+          * +20 outfit space (540 -> 560)
+          * +10 weapon capacity (240 -> 250)
+        * Leviathan:
+          * +120 mass (480 -> 600)
+          * +0.5 drag (7.6 -> 8.1)
+        * Marauder Falcon:
+          * +14 requires crew (43 -> 57)
+          * +16 bunks (80 -> 96)
+          * +20 cargo space (60 -> 80)
+          * +40 outfit space (580 -> 620)
+          * +10 weapon capacity (265 -> 275)
+        * Marauder Leviathan:
+          * +40 mass (640 -> 680)
+          * +0.5 drag (7.6 -> 8.1)
+      * Increased the sizes and damage of some human secondary weapons. (@Pointedstick)
+        * TODO: this
+      * Increased the outfit space of the Marauder Manta (390 -> 400). (@Pointedstick)
+      * Increased the outfit space and weapon capacity of the Rainmaker (230 -> 240 and 60 -> 70). (@Pointedstick)
+      * Buffed the Star Queen. (@Anarchist2)
+        * -1300000 base cost (5.5M -> 4.2M)
+        * -8 requires crew (41 -> 33)
+      * The Rano'ereks that appear as miners in human space are now better able to defend themselves. (@ravenshining)
+      * Made the battle in Terraforming Tundra less difficult. (@Anarchist2)
+      * Added a Small Biochemical Cell to the Summer Leaf. (@warp-core)
+      * Added some more Wanderer escort ships (equipped with Moonbeams) to "Wanderers: Sestor: Factory Escorts". (@warp-core)
+      * Significantly increase the energy capacity of Archons. (@Zitchas)
+      * Nerfed the flamethrower. (@Quantumshark)
+        * +100% firing heat (2 -> 4).
+        * -25% heat damage (200 -> 150)
+        * Added damage dropoff with range beyond 45 units. Damage is halved by maximum range.
+      * Increased the size of some Large Syndicate fleet variants. (@Quantumshark)
+      * Added a small amount of inertia reduction and force protection to Quarg engines. (@Zitchas)
+      * Made the Unfettered Hai attacking Wah Ki cowards and the Hai defending the system merciful, so fewer Unfettered die there. (@Hurleveur)
+      * Slightly buffed the Berserker and Hawk. (@Zitchas)
+        * Berserker:
+          * Mass: 110 -> 90
+        * Hawk:
+          * Mass: 150 -> 110
+          * Required crew: 2 -> 1
+      * Reduced the lifetime of the Hai Tracker from 600 to 475 (range reduced from 8400 to 6650). (@Zitchas)
+      * Buffed the Hai Wiliwaw Cooling. 570 -> 660 cooling. (@Zitchas)
+      * Buffed the Hai Grasshopper. (@Hurleveur)
+        * Outfit space: 169 -> 198
+        * Weapon capacity: 38 -> 47
+        * Some adjustments to outfitting and a new variant carrying an Ion Cannon.
+      * Adjusted the Quicksilver. (@Zitchas)
+        * Reduce chassic cost: 1090000 -> 836000
+        * Increased cargo space: 10 -> 15
+        * Removed the Cooling Ducts and use a cheaper generator.
+      * Reduce the 0.9.15 afterburner buff. (@Amazinite)
+        * Halved the buffs to thrust and fuel usage.
+        * Slightly nerfed the newer afterburners.
```

Comment:
The "half of what the buff was" wording that I used do be a little odd and I've seen at least one case of someone saying "afterburners are now half as strong," which isn't what I did. I'd instead just use the actual % change numbers, and compare them to the change that was made before.
```suggestion
        * Reduced thrust by 20-30% and increased fuel usage by 20-30% on all afterburners.
        * For reference, in 0.9.15, afterburners gained a 50% increase in thrust, 70-80% decrease in fuel usage, and 33% decrease in size.
```